### PR TITLE
23 create transaction object schema

### PR DESCRIPTION
**Pull request by mbrewer05**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+backend/.env
+backend/__pycache__
+backend/flaskMain.pyc
+backend/flaskMongo.pyc
```
Comment (ghuarng) [inline .gitignore:4]: Use this instead of old code to resolve merge conflict

File: backend/models/transaction.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "_id": "string",
+    "userId": "string",
+    "date": "string",
+    "value": signed_int,
```
Comment (ghuarng) [inline backend/models/transaction.json]: Resolve type error

File: backend/models/user.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "_id": "string",
+    "name": "string",
+    "username": "string",
+    "password": "string",
+    "budget": int,
```
Comment (ghuarng) [inline backend/models/user.json]: Resolve type errors

**Review by ghuarng — CHANGES_REQUESTED** [review]
Minor type errors in transaction.json and user.json

**Review by ghuarng — APPROVED** [review]

**Review by cpshan — APPROVED** [review]
Looks good to me

**Comment by ghost** [comment]
LGTM

**Review by mbrewer05 — COMMENTED** [review]
LGTM

**Pull request merged**